app/views/records/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="filters">
+  <%= render "shared/filters" %>
+</div>
+
+<div class="flexbox">
+  <div class="chart">
+    <%= bar_chart records_distance_per_month_path(@records_params), xtitle: "distance per month [km]", ytitle: @records_params[:group_by], thousands: "." %>
+  </div>
+  
+  <div class="chart">
+    <%= bar_chart records_distance_per_week_path(@records_params), xtitle: "distance per week [km]", ytitle: @records_params[:group_by], thousands: "." %>
+  </div>
+  
+  <div class="chart">
+    <%= bar_chart records_distance_path(@records_params), xtitle: "distance [km]", ytitle: @records_params[:group_by], thousands: "." %>
+  </div>
+</div>
+
+<div class="flexbox">
+  
+  <div class="chart">
+    <%= bar_chart records_average_pace_path(@records_params), xtitle: "average_pace [s]", ytitle: @records_params[:group_by], thousands: "." %>
```

Comment:
That's seconds per kilometer right?

Reply:
True.. need to be fixed

Reply:
Maybe a unit like `min/km` is a bit more common for this metric :thinking: 
It least my Garmin watch uses `min/km` as pace.

Reply:
Yes, but need to convert it to a decimal, so 6.5 instead of "6:30", as chartkick does not allow me to assign a different label to the value.. or i did not find it..

Reply:
Ah .. got it :+1:

Reply:
doing 6.5 min/km instead of 390 s/km now